src/System.IO.Pipelines/System/Buffers/MemoryPoolBlock.cs

```diff
@@ -1,147 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System.Text;
-using System.Threading;
-using System.Diagnostics;
-using System.IO.Pipelines;
-
-namespace System.Buffers
-{
-    /// <summary>
-    /// Block tracking object used by the byte buffer memory pool. A slab is a large allocation which is divided into smaller blocks. The
-    /// individual blocks are then treated as independent array segments.
-    /// </summary>
-    internal class MemoryPoolBlock : OwnedMemory<byte>
```

Comment:
Why don't we need this and MemoryPoolSlab anymore?

Reply:
We are not shipping this implementation in corefx, only in kestrel